python/cuml/dask/ensemble/base.py

```diff
@@ -264,6 +287,17 @@ def _func_fit(model, input_data, convert_dtype):
     return model.fit(X, y, convert_dtype)
 
 
+def _func_predict(model, input_data, **kwargs):
+    X = concatenate(input_data)
+    with cuml.using_output_type("numpy"):

```

Comment:
I'm not sure why this needs to be in numpy instead of gpu? Can't we return a cupy-style dask.array on gpu and call dask.array.mean on it?